fcrepo-http-api/src/main/java/org/fcrepo/http/api/FedoraLdp.java

```diff
@@ -356,6 +358,8 @@ public Response createOrReplaceObjectRdf(
                 }
                 throw new ClientErrorException("Invalid Content Type " + requestContentType, UNSUPPORTED_MEDIA_TYPE);
             }
+        } catch (final Exception e) {
```

Comment:
The question is, which exception are we catching here? `IOException` or `BinaryStoreException` or `could be anything`?